File: Client/src/app/models/message.ts
export interface Message {
    id: number;
    // The backend serializes "SenderID"/"ReceiverID" as "senderID"/"receiverID".
    senderID: string | null;
    // Some parts of the client may access the camelCase variant.
    senderId?: string | null;
    receiverID: string | null;
    receiverId?: string | null;
    content: string | null;
    createDate: string;
    isRead: boolean;
}

File: API/Model/Message.cs
using System;

namespace API.Model;

public class Message
{
    public int Id{get;set;}
    public string? SenderID{get;set;}
    public string? ReceiverID{get;set;}
    public string? Content{get;set;}
    public DateTime CreateDate{get;set;}
    public bool IsRead{get;set;}
    public AppUser? Sender{get;set;}
    public AppUser? Receiver{get;set;}
}
